main/src/main/java/cgeo/geocaching/ui/SimpleItemListView.java

```diff
@@ -327,13 +326,13 @@ private static Context wrap(final Context context) {
     private void init() {
         final Context ctw = getContext();
         inflate(ctw, R.layout.simpleitemlist_view, this);
-        binding = SimpleitemlistViewBinding.bind(this);
+        final SimpleitemlistViewBinding binding = SimpleitemlistViewBinding.bind(this);
         setOrientation(VERTICAL);
 
         listAdapter = new ItemListAdapter(binding.list);
         binding.list.setAdapter(listAdapter);
 
-        binding.listFilter.addTextChangedListener(ViewUtils.createSimpleWatcher(e -> triggerRefreshList()));
+        //binding.listFilter.addTextChangedListener(ViewUtils.createSimpleWatcher(e -> triggerRefreshList()));
```

Comment:
Way better than before, really like it!

Spotted a small leftover though... ;-)